frontend/src/components/BCDataGrid/BCGridBase.jsx

```diff
@@ -33,7 +33,6 @@ export const BCGridBase = forwardRef(({ autoSizeStrategy, ...props }, ref) => {
         suppressDragLeaveHidesColumns
         suppressMovableColumns
         suppressColumnMoveAnimation={false}
-        reactiveCustomComponents
```

Comment:
We have to keep this option to allow our inputs to commit values. Def not ideal but until we find a work around lets leave this here.

Reply:
That's odd, its marked as deprecated in their documentation as default on. Are we on an older version?

Reply:
we're running v32.0.2 and there is a v32.1.0
I have tested without this option and the inputs break so we'll have to investigate further. For this ticket lets just leave it in for now.

Reply:
Done, added back